Anlab.Mvc/Client/components/OrderForm.tsx

```diff
@@ -82,7 +82,7 @@ export default class OrderForm extends React.Component<undefined, IOrderState> {
         this.state = { ...initialState };
     }
     validate = () => {
-        const valid = this.state.quantity > 0 && !!this.state.project.trim();
+        const valid = this.state.quantity > 0 && this.state.quantity <= 100 && !!this.state.project.trim();

```

Comment:
@srkirkland I was asking @jpknoll about this. There are some frameworks we could import, but may not be worth it unless the form gets more complex.